js/parser.js

```diff
@@ -121,6 +121,7 @@
 
         // <sort>/<page>
         // sort and paging
+        var  pathString = this._path;
```

Comment:
This variable seems unnecessary, just pass `this._path` in two places that are using this.